core/domain/learner_progress_services.py

```diff
@@ -21,6 +21,7 @@
 from core.domain import subscription_services
 from core.domain import user_domain
 from core.platform import models
+from google.appengine.ext import ndb
```

Comment:
Nope, I meant create a new file in core/platform (see the other files there for examples of other GAE service boundaries that we put there). We try to separate the GAE-related stuff from the main domain logic, and it looks like we need a new "platform services" module for non-storage-model-specific datastore operations.